cmd/kubeadm/app/cmd/phases/workflow/runner.go

```diff
@@ -295,6 +295,10 @@ func (e *Runner) SetAdditionalFlags(fn func(*pflag.FlagSet)) {
 // command help, adding phase related flags and by adding phases subcommands
 // Please note that this command needs to be done once all the phases are added to the Runner.
 func (e *Runner) BindToCommand(cmd *cobra.Command) {
+	// keep track of the command triggering the runner
+	e.runCmd = cmd
```

Comment:
If we are moving this here, please remove it from L#396

Reply:
looks like i copied instead of moving.